src/AppStory/styles/_variables.scss

```diff
@@ -0,0 +1,9 @@
+@use 'sass:math';
+@import '~@gisatcz/ptr-core/lib/styles/_variables';
+@import '~@gisatcz/visat-components/lib/styles/_variables';
```

Comment:
Remove dependency on visat-components

Reply:
OK, but I am using its breakpoints, icons and tags. What should I do?